ctapipe/reco/event_classifier.py

```diff
@@ -3,6 +3,8 @@
 from astropy import units as u
 
 from sklearn.ensemble import RandomForestClassifier
+from sklearn.neural_network import MLPClassifier
+from sklearn.preprocessing import StandardScaler
 
```

Comment:
For `MLPClassifier` I'd say yes. Testing at least two different classifiers is probably a good idea anyway to show that the modularity works.

For `StandardScaler` I suggest to import it there where you add the functionality that uses it. So I'd say in `RegressorClassifierBase` since `MLPRegressor` is also something we can look into.